apps/sms/services/test/unit/mock_drafts.js

```diff
@@ -18,4 +18,7 @@ var MockDrafts = {
 
 function MockDraft(draft) {
   Object.assign(this, draft);
+  if (!this.id) {
+    this.id = 'draftId';
```

Comment:
The real Draft is assigning an id if there is not any in the parameters, so I thought it would be nice to do the same in the mock. Is it too much logic ?